### PR TITLE
Add CopyManga Setting Change Title to Simplified Chinese Optionally

### PR DESCRIPTION
**Pull request by rinshan-kaihou**
for library gobal search
search manga title with the same text

添加标题的繁体转简体的设定选项
方便在Library中的搜索，
简体和繁体混在一起不太好搜索

### CODE REVIEW

File: src/zh/copymanga/build.gradle [hunk 1]
```diff
@@ -5,8 +5,13 @@ ext {
...
-    extVersionCode = 3
+    extVersionCode = 4
     libVersion = '1.2'
 }
-
 apply from: "$rootDir/common.gradle"
+
+dependencies {
+    compileOnly project(':duktape-stub')
+    implementation 'org.apache.commons:commons-text:1.8'
```
Comment (arkon) [inline src/zh/copymanga/build.gradle:15]: Do you need this?

Comment (CarlosEsco) [reply src/zh/copymanga/build.gradle:15]: It doesn't seem to be used

Comment (rinshan-kaihou) [reply src/zh/copymanga/build.gradle:15]: need not
I forgot to delete this

Comment (CarlosEsco) [reply src/zh/copymanga/build.gradle:15]: can you open a pull request and remove it.  Don't change the gradle version, no reason to do a new release

**Pull request merged**